Makefile

```diff
@@ -265,7 +265,7 @@ build-push-container-multiplatform-dev: REPO_URL = "$(OPENSOURCE_REPO_URL_DEV)"
 build-push-container-multiplatform-dev: build-push-container-multiplatform
 
 .PHONY: push-container-manifest-dev
-push-container-manifest-dev: REPO_URL="$(OPENSOURCE_REPO_URL_DEV)"
+push-container-manifest-dev: REPO_URL = "$(OPENSOURCE_REPO_URL_DEV)"
```

Comment:
:hear_no_evil:

Reply:
This doesn't actually matter, but makes it clear this isn't bash assignment.